src/test/java/org/symphonyoss/symphony/messageml/elements/form/DateSelectorTest.java

```diff
@@ -15,64 +15,75 @@
   
   @Test
   public void sendValidDateSelectorOnPresentationML() throws Exception {
-    context.parseMessageML("<messageML><form id=\"" + FORM_ID_ATTR + "\"><div class=\"date-selector\"/></form></messageML>", null, MessageML.MESSAGEML_VERSION);
-    assertDataFromValidParsedTag();
+    context.parseMessageML("<messageML><form id=\"" + FORM_ID_ATTR + "\"><div class=\"date-selector\" name=\"some-name\"/></form></messageML>", null, MessageML.MESSAGEML_VERSION);
```

Comment:
This is not correct. When sending as the div, the attribute should be "data-name" and not "name" because it should represent what the presentationML parsed will be.